pkg/controllers/securitypolicy_controller.go

```diff
@@ -217,6 +231,16 @@ func (r *SecurityPolicyReconciler) setupWithManager(mgr ctrl.Manager) error {
 			controller.Options{
 				MaxConcurrentReconciles: runtime.NumCPU(),
 			}).
+		Watches(
+			&source.Kind{Type: &v1.Namespace{}},
```

Comment:
May I know why we need to watch namespace event for named port?

Reply:
We have namespaceSelector
`   destinations:
        - podSelector:
            matchLabels:
              role: db
            matchExpressions:
              - {key: k1, operator: In, values: [a1,a2]}
              - {key: k2, operator: NotIn, values: [a3,a4]}
        - namespaceSelector:
            matchLabels:
              role: db`
, if one namespace's label satisfied the criteria for the selector, we should reconcile.